src/Pages/EmailPreferences/EmailPreferences.js

```diff
@@ -0,0 +1,152 @@
+import './EmailPreferences.css';
+
+export default function EmailPreferencesPage(props) {
+  const BLUE = '#2d75eb';
+  const GREEN = '#1cc75b';
+  const RED = '#cf533a';
+  const [isOptedIntoEmails, setIsOptedIntoEmails] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [errorLoading, setErrorLoading] = useState(false);
+  const [userEmail, setUserEmail] = useState('');
+  const [buttonColor, setButtonColor] = useState(BLUE);
+  const [userFirstName, setUserFirstName] = useState('');
+  const [userLastName, setUserLastName] = useState('');
+  const [message, setMessage] = useState('Submit');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const urlParams = new URLSearchParams(window.location.search);
+      const userEmailParam = urlParams.get('user');
+      setUserEmail(userEmailParam);
+      if (!userEmailParam) {
+        window.location = window.location.origin;
+      } else {
+        const response = await getUserData(urlParams.get('user'));
+        if (!response.error) {
+          setUserFirstName(response.responseData.firstName);
+          setUserLastName(response.responseData.lastName);
+          setIsOptedIntoEmails(response.responseData.emailOptIn);
+        } else {
+          setErrorLoading(true);
+        }
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  const handleSave = async () => {
+    const response = await setUserEmailPreference(
+      userEmail,
+      isOptedIntoEmails
+    );
+    if (!response.error) {
+      setMessage('Yay! You changed your email preference!');
+      setButtonColor(GREEN);
+      setTimeout(() => {
+        setButtonColor(BLUE);
+        setMessage('Submit');
+      }, 3000);
+    } else {
+      setButtonColor(RED);
+      setMessage('Unable to submit your preference, try reloading? :(');
+    }
+  };
+
+  if (errorLoading) {
+    return (
+      <div className='email-preferences'>
+        {/*
+          dumb hacks because the navbar otherwise covers up the below content
+        */}
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <Container
+          style={{
+            textAlign: 'center',
+          }}
+        >
+          <h3>
+            The email page was unable to load at this time.
+          </h3>
+          <h4>
+
+            If the problem persists after reloading this page, please contact us on Discord
+            (<a href='https://discord.gg/KZCKCEz5YA'>invite link</a>)
+          </h4>
+        </Container>
+      </div>
+    );
+  }
+
+  return (
+    <div className='email-preferences'>
+      <Header title='Email Preferences' />
+      <Container
+        style={{
+          textAlign: 'center',
+        }}
+      >
+        {!loading && <>
+          <h1>
+            Hello {userFirstName} {userLastName},
+          </h1>
+          <p>
+            What would you like to hear about?
```

Comment:
would be sick if this was a little bit bigger.

right now it gets overshadowed by the header above and users might skip this to reading the radio groups below.

maybe could also change this text to something like: "Do you want to keep receiving club emails and updates?"